spec/system/vendor_api/vendor_receives_application_spec.rb

```diff
@@ -53,7 +53,7 @@ def then_it_should_include_the_data_from_the_application_form
       id: @provider.application_choices.first.id.to_s,
       type: 'application',
       attributes: {
-        personal_statement: nil,
+        personal_statement: 'Why do you want to become a teacher?: I WANT I WANT I WANT I WANT What is your subject knowledge?: Everything',
```

Comment:
Should this have line breaks to separate questions?

Reply:
Sounds good 👍 
I'll add a line break.

Reply:
Updated in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/700/commits/25d0ed3d5243f4f502ce59bf85f3726dee10006a. :)